### ISSUE TITLE
Not working for some reason :( 

### ISSUE THREAD

**Issue by koleror** [pitomba/spriter#28]
Hi,
I just finished implementing your library but for some reason I can't get the images to work...
When I add the right class to a span, it matches with the right css class the the image doesn't show up.
When I look at the css, I'm a bit surprised because you do not specify any image height or width. Only a position is set. How can the browser know what size the image is?
Here is what I put in the HTML:

``` html
<span class="mytag sreceipt"></span>
```

This is the matching css:

``` css
.mytag.sreceipt {
background-position: -6358px 0px;
}
.mytag {
background: url("../images/sprite/sprite.png");
}
```

I checked and the sprite is loaded properly (no 404).

Please answer asap or I'll have to use another library :( 

Thanks!

**Comment by koleror** [comment]
To make it work, I need to add this:

``` css
.mytag.sreceipt {
background-position: -6358px 0px;
width: 138px;
height: 188px;
}
.mytag {
background: url("../images/sprite/sprite.png");
display: inline-block;
}
```

**Comment by koleror** [comment]
I fixed the problem and made a pull request :)

**Comment by victorpantoja** [comment]
Hi  Hugo

Thank you so much about your contribution. Please, give me some time to
arrive work and get things fixed.

If you are interesting in knowing another use case, check out
globoesporte.com. You can see that our menu have lot's of Brazilian teams
flags generated by spriter. As we receive more than 8 millions visits per
day, we are sure spriter works well in such case.

Let us know if you need some help implementing your code with spriter and
if you find another bug :)

Best,
Victor

**Comment by koleror** [comment]
Hi Victor,

Hmm that is weird I don't get how it works on your website and why it is not on mine without my patch...
Any idea?

**Comment by romulojales** [comment]
Hi @koleror ,

The sprite's responsibility is to provide a sprite image and it's positional's css.  Any other feature is client responsibility.

I think the item demission is a critical property to have mainly to avoid image overflow in the box, so you're correct when you say that we need to provide the width and height of the sprite item. The sprites knows that properties and it can be write into the css to provide the right demission.

About the display: inline-block, I took a look and in our website we wrote this property by hand, in the component's css. So I think we can use too.

I think it's those are a great features because we can build mixed items sized sprites.

You have mentioned that you made a pull request, where is it?
